configs/auto-sgen-aotjit-serial-binary.conf

```diff
@@ -0,0 +1,18 @@
+{
+  "Name": "auto-sgen-aotjit-serial-binary",
+  "Count": 10,
+  "Mono": "$ROOT/bin/mono-sgen",
+  "AOTOptions": [
+    "-O=all",
+    "--aot"
```

Comment:
well yeah, but we might want to fine-tune at some point and the way to do that is with `--aot[=options]`